src/TypeSpec.Extension/Emitter.Csharp/test/Unit/modelType.test.ts

```diff
@@ -76,7 +76,6 @@ op test(@body input: Pet): Pet;
                         Kind: "String",
                         IsNullable: false
                     },
-                    IsNullable: false,
```

Comment:
this is removed because we never have a `IsNullable` property in the type `InputModelProperty`.